docs-src/security-model.md

```diff
@@ -72,14 +72,12 @@ readable by most roles, so instead we'll focus on write access for this model.
 The extra configuration options are not possible to control with RBAC. Instead,
 they will be controlled with configuration fields on GatewayClasses:
 
-* **allowedGatewayNamespaceSelector**: This field is a selector of namespaces
-  that Gateways can use this GatewayClass from. This is a standard Kubernetes
-  LabelSelector, a label query over a set of resources. The result of
-  matchLabels and matchExpressions are ANDed. Controllers must not support
-  Gateways in namespaces outside this selector. An empty selector (default)
-  indicates that Gateways can use this GatewayClass from any namespace. This
-  field is intentionally not a pointer because the nil behavior (no namespaces)
-  is undesirable here.
+* **allowedGatewayNamespaceSelector**: AllowedGatewayNamespaceSelector is a
```

Comment:
Out of date. This should be `allowedGatewayNamespaces`.

Reply:
+1

Reply:
Updated, thanks!